guides/advanced-use/project-layouts.md

```diff
@@ -15,9 +15,13 @@ app
 |   └── post.js
 ├── routes
 |   └── post.js
-└── templates
-    ├──components/tags.hbs
-    └── post.hbs
+├── templates
+|   ├──components/tags.hbs
```

Comment:
Looks like this one might be missing a space? All of the others have a space after the last `-`

Reply:
Yep missed that in original PR - fixed